projects/plugins/jetpack/extensions/plugins/publicize/components/panel/index.js

```diff
@@ -30,11 +29,9 @@ const PublicizePanel = ( { prePublish } ) => {
 		isRePublicizeFeatureEnabled,
```

Comment:
Does it make sense to still have this boolean? It's used in the [Publicize Form](https://github.com/Automattic/jetpack/blob/trunk/projects/js-packages/publicize-components/src/components/form/index.js) but we can probably get rid of the usage there as well.

Reply:
I think the publicize module can be disabled, which is why I think this exists. Let me experiment with that and reply to this comment.

Reply:
This had nothing to do with that, I have removed this boolean